dependencies/vaticle/repositories.bzl

```diff
@@ -20,8 +20,8 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 def vaticle_dependencies():
     git_repository(
         name = "vaticle_dependencies",
-        remote = "https://github.com/vaticle/dependencies",
-        commit = "a8b3b714a5e0562d41f4c05ca8f266d48b7d0fd3", # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_dependencies
+        remote = "https://github.com/jamesreprise/dependencies",
+        commit = "a0ddfa6d97a5e124db2190236de4d0c0162ae9bc", # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_dependencies
```

Comment:
TEMPORARY: whilst we investigate the issue for running tests in Factory, we need to use a fork with the updated dependencies.